lookup/api/views.py

```diff
@@ -74,10 +74,18 @@ def partial_update(self, request, id=None):
         self._visitors_data.updateSerializer(save=False)
 
         instance = self._visitors_data.getQueryset().get(pk=id)
-        data = {'name': request.data['name'], 'visits_count': instance.visits_count}
+        data = {'name': request.data['name'], 'visits_count': instance.visits_count, 'updated_at': timezone.now()}
         self._visitors_data.updateSerializer(instance=instance, data=data, partial=True)
         print(f"updated a user's name of id-{instance.id} to {request.data['name']}")
         return Response({"status": "Success"}, status=status.HTTP_206_PARTIAL_CONTENT)
         
     def destroy(self, request, id=None):
-        pass
\ No newline at end of file
+        self._visitors_data.updateQueryset()
```

Comment:
implemented destro method to delete the requested visitor's data in db sent with delete method.